tests/ld-key-pair.spec.js

```diff
@@ -0,0 +1,69 @@
+'use strict';
+
+const chai = require('chai');
+chai.use(require('dirty-chai'));
+chai.should();
```

Comment:
I see that the `should` api is not actually being used.  So maybe it should be removed? If it stays, it's good to define globals in the eslintrc file in the test folder like:

https://github.com/digitalbazaar/bedrock-ledger-node/blob/master/test/mocha/.eslintrc

The `assertNoError` global there is a bedrock thing, so should not be included here.

Reply:
Ah, for some reason, I thought `expect` also required a `should()` invocation. Removed it.